mlflow/pyfunc/model.py

```diff
@@ -270,6 +205,21 @@ def _load_pyfunc(model_path):
                     dst_dir=saved_artifact_name))
         artifacts[saved_artifact_name] = tmp_artifact_path
 
-    context = PythonModelContext(artifacts=artifacts, directory_managers=directory_managers)
+    context = PythonModelContext(artifacts=artifacts)
     python_model.load_context(context=context)
-    return python_model
+    return _PythonModelPyfuncWrapper(python_model=python_model, context=context)
+
+
+class _PythonModelPyfuncWrapper:
```

Comment:
Add a class doc saying which interface this is extending

Reply:
Added the same class doc in used by other wrappers (ex: the one in the `mlflow.pytorch` module)